third_party/crt/repos.bzl

```diff
@@ -15,7 +15,8 @@ def crt_repos(local = None):
         maybe(
             http_archive,
             name = "crt",
-            url = "https://github.com/lowRISC/crt/archive/refs/tags/v0.4.6.tar.gz",
-            strip_prefix = "crt-0.4.6",
-            sha256 = "b1dee52e96b18a4a2e180be18f2fbb65a918d5e4350c9b664883b79658f711ec",
+            url = "https://github.com/lowRISC/crt/zipball/pull/27/head",
+            type = "zip",
+            strip_prefix = "lowRISC-crt-470c151",
+            sha256 = "27a049a15391a9369cb8de1b874392b518ade0e6c8a37630642531b9ae027fc5",
```

Comment:
You really need to create a CRT release.  You can create a release using the "Create Release" action under the "Actions" tab.  I may need to add you as a collaborator to all you permission to create the release - let me know if you can't trigger the Create Release action.

Reply:
Yes, this was just to make sure that the change was working since it's quite hard to reproduce all tests done in the CI. I'll try to create a release tomorrow.